game-using-jsDOM/script.js

```diff
@@ -0,0 +1,219 @@
+// Array of words that we will use for the game 
+const words = ['penguins', 'elephant', 'koala', 'synonym', 'peanut',
+    'english', 'ruby', 'react', 'friday', 'airplane', 'banana', 'birthday',
+    'waterfall', 'trampoline', 'window', 'president', 'basketball', 'hiccup',
+    'cheerleader', 'kangaroo', 'groundhog', 'soccer', 'handball', 'caterpillar',
+    'shoelaces', 'hungry', 'fishing', 'monster'];
+
+/**
+ * Gets a random word from the words array declared on line 2
+ * @returns a word that has been randomly selected from the words_array
+ */
+function getWord() {
+    return (words[Math.floor(Math.random() * words.length)]);
+}
+
+// grab and store the body element tag
+let body = document.getElementsByTagName("body")[0];
```

Comment:
Would you mind using `document.body`? The best way to get body element is by using `document.body`

Reply:
I did not know you could do document.body, I have since changed it. Thank you!!